plugins/repository-azure/src/internalClusterTest/java/org/opensearch/repositories/azure/AzureBlobStoreRepositoryTests.java

```diff
@@ -188,6 +190,7 @@ protected String requestUniqueId(final HttpExchange exchange) {
     @SuppressForbidden(reason = "this test uses a HttpServer to emulate an Azure endpoint")
     private static class AzureHTTPStatsCollectorHandler extends HttpStatsCollectorHandler {
 
+        private static final Logger testLogger = LogManager.getLogger("AzureHTTPStatsCollectorHandler");
```

Comment:
nit: Logger init can be `LogManager.getLogger(AzureHTTPStatsCollectorHandler.class);` to keep traceability in case of class name changes.

Reply:
Sure I like that!